utils/run_tests.sh

```diff
@@ -7,11 +7,13 @@ find vumi/ -name '*.pyc' -delete
 echo "=== Erasing previous coverage data..."
 coverage erase
 echo "=== Running trial tests..."
-coverage run `which trial` --reporter=subunit vumi | subunit-1to2 | tee results.txt | subunit2pyunit
+# only check the module that we care about
+coverage run `which trial` --reporter=subunit vumi.transports.smpp | subunit-1to2 | tee results.txt | subunit2pyunit
```

Comment:
We use more than just this, right?

https://github.com/hnec-vr/vumi-libya/blob/develop/hnec/apps.py

Reply:
Heh... I should have said 'module that **I** care about' (for the purposes of this change. But I'll revert the change because, you're right, we do use more than the transports. But please don't ask me to fix the 200ish test failures :smile: